### PR TITLE
Alerts UI docs with cleanup instructions

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/setup-robusta/alertsui.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Alerts UI
+######################
+
+With Robusta Alerts UI, you can create and customize Prometheus Rules using templates, without needing to know PromQL.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:4]: With **the** Robusta Alerts UI

File: docs/setup-robusta/alertsui.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.. image:: /images/managed-alerts-ui.png
+
+
+
+This guide covers enabling managed alerts with Robusta UI. This involves two steps: One to enable the Alert Rules on the UI and another to disable default Prometheus alerts installed by Kube Prometheus Stack.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:10]: with *the* Robusta UI

Comment (aantn) [reply docs/setup-robusta/alertsui.rst:10]: Can you use bullet points for the two steps?

File: docs/setup-robusta/alertsui.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+.. note::
+
+    Alerts UI works only with Prometheus installed by Robusta. :ref:`Embedded Prometheus Stack`
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:14]: I suggest rephrasing as: 

`To use the Alerts UI, you must use Robusta's embedded Prometheus stack.`

File: docs/setup-robusta/alertsui.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Enable Alerts UI
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:16]: Enable *the* Alerts UI

File: docs/setup-robusta/alertsui.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+--------------------
+
+To start seeing all the alerts and customize them, we first need to enable Alert Rules in the Robusta UI.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:19]: To see alerts and customize them, you must first enable Alert Rules in the Robusta UI.

File: docs/setup-robusta/alertsui.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+1. Go to `platform.robusta.dev <https://platform.robusta.dev/>`_ -> Alerting -> Alerts
+2. Click "Enable Alert Rules"
+
+.. image:: /images/click-enable-alert-rules.png
+
+
+
+Enable Robusta Managed Alerts
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:28]: I'm a little confused by "Enable Alerts UI" vs "Enable Alert Rules" vs "Enable Robusta Managed Alerts"

They all sound like the same thing to me. Can we be more clear what each one is doing? 

(In your instructions, refer to the exact text on the buttons. E.g. you can write `Click the "Enable Alert Rules" button.` But in your titles and description of what is happening, try to make sense of it.)

File: docs/setup-robusta/alertsui.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+********************************************
+
+Next we need to disable default Prometheus alerts from Kube Prometheus Stack and enable Robusta managed alerts. Add the following config to your ``generated_values.yaml``.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:31]: Why? (To avoid duplicates. Explain it.)

File: docs/setup-robusta/alertsui.rst [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+        prometheus: false
+        prometheusOperator: false
+
+Then do a :ref:`Helm Upgrade <Simple Upgrade>`.
+
+
+Disable Robusta Managed Alerts
+---------------------------------
+
+To go back to Kube Prometheus Satck built in alerts, remove the config added in the previous step from your ``generated_values.yaml``. Then do a :ref:`Helm Upgrade <Simple Upgrade>`.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:63]: typo in Stack

File: docs/setup-robusta/alertsui.rst [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+Cleanup
+*******************
+
+Robusta stores its managed rules in PrometheusRules CR's that start with ``robusta-prometheus.rules--``. If left in the cluster, you might have double alerts.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:68]: CRs not CR's. But I would spell out "custom resource" explicitly E.g. "in custom resources named PrometheusRule..."

File: docs/setup-robusta/alertsui.rst [hunk 10]
```diff
@@ -0,0 +1,78 @@
+Alerts UI
+######################
+
+With the Robusta Alerts UI, you can create and customize Prometheus Rules using templates, without needing to know PromQL.
+
+.. image:: /images/managed-alerts-ui.png
+
+This guide covers enabling managed alerts with the Robusta UI. This involves two steps:
+
+* Enable the Alert UI on the Robusta UI.
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:10]: Please use the same text as below ("Active Alerts Interface in the Robusta UI")

File: docs/setup-robusta/alertsui.rst [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+            prometheus: false
+            prometheusOperator: false
+
+.. code-annotations::
+    1. Creates Robusta managed alerts in your cluster.
+
+Then do a :ref:`Helm Upgrade <Simple Upgrade>`.
+
+
+Disable Robusta Managed Alerts
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:64]: This section seems similar to the above (`Disable Default Alert Rules`) so it's easy for me to get confused and think this is a step you need to do in order to enable the feature.

Can you:

1. Change the title to something more clear ("Disabling the Feature")
2. Start with text like , "If you choose to stop using the Robusta alerts UI, you can do so at any time and go back to using..."

File: docs/setup-robusta/alertsui.rst [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+---------------------------------
+
+To go back to using Kube Prometheus Stack built in alerts, remove the config added in the previous step from your ``generated_values.yaml``. Then do a :ref:`Helm Upgrade <Simple Upgrade>`.
+
+Cleanup
```
Comment (aantn) [inline docs/setup-robusta/alertsui.rst:69]: This is part of the previous section. I don't think it needs a separate subheader as that implies its something different to me.

File: docs/images/managed-alerts-ui.png
[the remark this answers is not in the document]
Comment (pavangudiwada) [reply docs/images/managed-alerts-ui.png:1]: :heavy_check_mark: Done

**Pull request merged**